docker/entrypoint-uwsgi.sh

```diff
@@ -9,4 +9,5 @@ exec uwsgi \
   --processes ${DD_UWSGI_NUM_OF_PROCESSES:-2} \
   --threads ${DD_UWSGI_NUM_OF_THREADS:-2} \
   --wsgi dojo.wsgi:application \
-  --buffer-size="${DD_UWSGI_BUFFER_SIZE:-4096}"
+  --buffer-size="${DD_UWSGI_BUFFER_SIZE:-4096}" \
+  --http 0.0.0.0:8081 --http-to ${DD_UWSGI_ENDPOINT}
```

Comment:
is that opening the possibility to acess uwsgi directly without going through nginx? doesn't that have security implications? it's my understanding that nginx works as a kind of firewall in front of uwsgi, and for example won't let any headers going through but the ones that are allowed as per wsgi_params file.

Reply:
could this one go through nginx too without opening anything new on uwsgi?

Reply:
Unless you don't expose the port in docker-compose or Kubernetes, it should not be an issue. I don't know if there is a better way to do it. I found some tools like this one: https://github.com/andreif/uwsgi-tools but I'm not sure it's a good idea to integrate that in image.

Reply:
actually going through nginx would be a bad idea because if nginx fails we would restart uwsgi for no reason. 

you're right it's not a big risk as the port is not exposed, and this script is not used outside of docker-compose/ kubernetes context. 

you're right too, it's not trivial to check a uwsgi server state; the socket endpoint is listening on port 3031 but we can't do much without additional tooling. just tried curl for example isn't really helpful
```
curl localhost:3031
curl: (52) Empty reply from server
```

 you probably came across this article :) https://stackoverflow.com/questions/57122352/how-to-check-uwsgi-with-gevent-at-socket-file-in-k8s-readinessprobe-and-liveness

you're all good for this comment then; could you just add a comment in the entrypoint script that explains that this `--http ` option is required for the k8s uwsgi livenessProbe configured in django-deployment.yaml ? 

thanks!

Reply:
Done as well!

Reply:
thanks :)